ApiBundle/Transformer/NodeGroupRoleTransformer.php

```diff
@@ -52,25 +66,43 @@ public function transform($nodeGroupRole)
     }
 
     /**
-     * @param FacadeInterface             $facade
-     * @param NodeGroupRoleInterface|null $source
+     * @param FacadeInterface     $nodeRoleFacade
+     * @param GroupInterface|null $group
      *
      * @throws RoleNotFoundHttpException
-     * @return mixed
+     * @throws TransformerParameterTypeException
+     *
+     * @return null|NodeGroupRoleInterface
      */
-    public function reverseTransform(FacadeInterface $facade, $source = null)
+    public function reverseTransform(FacadeInterface $nodeRoleFacade, $group = null)
```

Comment:
https://github.com/open-orchestra/open-orchestra-cms-bundle/blob/master/ApiBundle/Transformer/AreaTransformer.php#L223

Regarding what is done in others transformers (like the node one).e

I would prefer to let three parameters.

If an integrator will override it, i will have to respect the signature given by the class overrided